src/vm/zapsig.cpp

```diff
@@ -1127,7 +1127,7 @@ BOOL ZapSig::EncodeMethod(
     TypeHandle ownerType;
 
 #ifdef FEATURE_READYTORUN_COMPILER
-    if (IsReadyToRunCompilation())
+    if (IsReadyToRunCompilation() && !pMethod->GetModule()->IsInCurrentVersionBubble())
```

Comment:
> && !pMethod->GetModule()->IsInCurrentVersionBubble() [](start = 33, length = 53)

This is a meaningful behavior change even when this feature isn't enabled.

Reply:
This was absolutely correct and the check was making a functional difference. Changed!